card-1/css/_global.scss

```
@@ -0,0 +1,27 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap');
+
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+@mixin flex($direction){
+    display: flex;
+    flex-direction: $direction;
+}
+
+$dark-cyan:hsl(185, 75%, 39%);
```

```suggestion
$dark-cyan: hsl(185, 75%, 39%);
```
pregledaj sve fajlove